what_the_fec/api/columns_information.py

```diff
@@ -0,0 +1,22 @@
+def get_columns_information_query(table_name):
+    return f"""
+        SELECT 
+            COLUMN_NAME, 
+            DATA_TYPE, 
+            CHARACTER_MAXIMUM_LENGTH, 
+            COLUMN_DEFAULT, 
+            IS_NULLABLE 
+            FROM INFORMATION_SCHEMA.COLUMNS
+            WHERE table_name = '{table_name}' AND COLUMN_NAME != "id"
+        """
```

Comment:
we should do our best not to f string table_name in and parametrize the query later after putting the placeholder for table_name. I know we have some original ones that are like this that we need to cleanup I just hope that number does not grow.

Reply:
also pulling this sql into its own functions is a great call it is in my plan to pull some of this down into the storage layer if time permits. worst case I will likely pull it into functions in the api(soon to be renamed to routes) package.

Reply:
Per our chat, let's put this in a TODO for the future. I'll add a TODO note.